src/VaultFactory.sol

```diff
@@ -59,8 +59,8 @@ contract VaultFactory {
    */
   function deployVault(
     IERC20 _asset,
-    string memory _name,
-    string memory _symbol,
+    string calldata _name,
+    string calldata _symbol,
```

Comment:
I don't think this changes anything since it gets copied to memory in the bytecode anyway on line 72.
```suggestion
    string memory _name,
    string memory _symbol,
```

Reply:
Tested it out and it uses about 4k more gas to use `calldata` here instead of `memory`.

Reply:
Ah thanks. I should have checked it out.

Reply:
> Ah thanks. I should have tested it out instead of blindly trusting auditors.

Reply:
Reverted in this commit: https://github.com/GenerationSoftware/pt-v5-vault/pull/45/commits/1d8735b5b754a8698ad4cd302a4b2f9fb56eb45b